sw/device/sca/aes_serial/aes_serial.c

```diff
@@ -0,0 +1,334 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/aes.h"
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/base/log.h"
+#include "sw/device/lib/dif/dif_gpio.h"
+#include "sw/device/lib/dif/dif_rv_timer.h"
+#include "sw/device/lib/handler.h"
+#include "sw/device/lib/irq.h"
+#include "sw/device/lib/pinmux.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "sw/device/lib/uart.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+static dif_gpio_t gpio;
+static dif_rv_timer_t timer;
+
+// UART input maximum buffer sizes
+static const uint32_t kMaxInputLengthText = 128;
+static const uint32_t kMaxInputLengthBin = 64;
+
+// AES Configuration parameters. These values must match the test driving
+// side.
+static const uint32_t kAesBlockSize = 16;
+static const uint32_t kAesKeySize = 16;
+
+// Simple serial status codes
+typedef enum simple_serial_result {
+  kSimpleSerialOk = 0,
+  kSimpleSerialError = 1
+} simple_serial_result_t;
+
+// Simple serial protocol version 1.1
+static const uint32_t kSimpleSerialProtocolVersion = 1;
+
+// GPIO capture trigger values
+static const uint32_t kGpioCaptureTriggerH = 0x08200;
+static const uint32_t kGpioCaptureTriggerL = 0x00000;
```

Comment:
Would you mind `s/H/High/` and `s/L/Low/`?

Reply:
Done.